nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -7,12 +7,12 @@
   <BehaviorTree ID="MainTree">
     <RecoveryNode number_of_retries="6" name="NavigateRecovery">
       <PipelineSequence name="NavigateWithReplanning">
-        <RateController hz="1.0">
+        <SpeedController min_rate="0.1" max_rate="1.0" min_speed="0.0" max_speed="0.25">
```

Comment:
I don't think we should be updating these BTs with this. Lets just add the controller and we can discuss the right default trees / nodes we want to use in another ticket.